sdk/cosmosdb/cosmos/src/client/Item/Item.ts

```diff
@@ -178,6 +188,29 @@ export class Item {
    *
    * @param body - The definition to replace the existing {@link Item}'s definition with.
    * @param options - Additional options for the request
+   * @example
+   * ```ts snippet:ItemReplace
+   * import { CosmosClient } from "@azure/cosmos";
+   *
+   * const endpoint = "https://your-account.documents.azure.com";
+   * const key = "<database account masterkey>";
+   * const client = new CosmosClient({ endpoint, key });
+   *
+   * const { database } = await client.databases.createIfNotExists({ id: "Test Database" });
+   *
+   * const { container } = await database.containers.createIfNotExists({ id: "Test Container" });
+   *
+   * interface TodoItem {
+   *   title: string;
+   *   done: boolean;
+   *   id: string;
+   * }
+   *
+   * const { resource: item } = await container.item("id").read<TodoItem>();
```

Comment:
it needs to be .item("id", pkValue).read(). 
Pk Value is required for read call